code/mob/living/carbon/human.dm

```diff
@@ -3648,3 +3650,43 @@
 			src.health_mon.icon_state = "10"
 		if (-INFINITY to 0)
 			src.health_mon.icon_state = "0"
+
+/mob/living/carbon/human/proc/update_arrest_icon()
+	if (!src.arrestIcon)
+		return
+
+	var/arrestState = ""
+	var/visibleName = src.face_visible() ? src.real_name : src.name
+	var/datum/db_record/record = data_core.security.find_record("name", visibleName)
+	if(record)
+		var/criminal = record["criminal"]
+		if(criminal == "*Arrest*" || criminal == "Parolled" || criminal == "Incarcerated" || criminal == "Released" || criminal == "Clown")
+			arrestState = criminal
+	else if(src.traitHolder.hasTrait("stowaway") && src.traitHolder.hasTrait("jailbird"))
+		arrestState = "*Arrest*"
```

Comment:
these should be defines, like `#define STATE_CRIMINAL_ARREST "*Arrest*"` or something